pages/learn/index.tsx

```diff
@@ -90,6 +93,24 @@ const LearnPage: CustomPage<{
           transition={transition(DURATION.medium4, EASING.standard)}
         >
           <Header>{t("schedule")}</Header>
+          <TextGlance
```

Comment:
Text Glances must only be put on the top of the home page. I suggest you place this Glance in [Home Layout](https://github.com/suankularb-wittayalai-school/mysk-frontend/blob/33a73614f2d3414f5ab8074778aef8adbdf81c4b/components/home/HomeLayout.tsx), before `{children}`.

https://github.com/suankularb-wittayalai-school/mysk-frontend/blob/33a73614f2d3414f5ab8074778aef8adbdf81c4b/components/home/HomeLayout.tsx#L29-L32